### DISCUSSION TITLE
Update Stats Only

### DISCUSSION THREAD

**Discussion by Nehamnsu** [olahallengren/sql-server-maintenance-solution#513] [Questions & Answers] [answered]
Hi,

I am a big fan of IndexOptimize SP and am using it for our current PROD server. I have a question regarding making it work only for updating stats.

By default, index optimize does index maintenance as well. I am just looking to update statistics for my DB. Is it possible to just update stats without doing any index maintenace? Would this work?
Please advise

 EXECUTE [dbo].[IndexOptimize]    @Databases = 'USER_DATABASES' ,    @FragmentationLow = NULL ,    @FragmentationMedium = NULL ,    @FragmentationHigh = NULL ,    @UpdateStatistics = 'ALL' ,    @OnlyModifiedStatistics = N'Y' ,    @LogToTable = N'Y';

**Comment by Stan-C421** [comment, marked as answer]
Hi,
From the documentation yes, the above should only update the modified Index and Column statistics of all user databases. Just give it a run on a test server and check the log table.
[https://ola.hallengren.com/sql-server-index-and-statistics-maintenance.html](url)

**FragmentationLow**
NULL | Do not perform index maintenance. This is the default for a low-fragmented index.

**FragmentationMedium**
NULL | Do not perform index maintenance.

**FragmentationHigh**
NULL | Do not perform index maintenance.

-ciao

**Reply by Nehamnsu** [reply]
thank you
